README.md

```diff
@@ -56,10 +152,184 @@ These three files can be downloaded directly from `inputs` and `dataset` directo
 
 Refer to [installation section](#installation) to set-up DeezyMatch on your local machine. 
 
-:warning: In the following tutorials, we assume a directory structure specified in the [this section](#data-and-directory-structure-in-tutorials).
+:warning: In the following tutorials, we assume a directory structure specified in [this section](#data-and-directory-structure-in-tutorials).
  
  Written in the Python programming language, DeezyMatch can be used as a stand-alone command-line tool or can be integrated as a module with other Python codes. In what follows, we describe DeezyMatch's functionalities in different examples and by providing both command lines and python modules syntaxes.
 
+### Quick tour 
+
+In this "quick tour", we go through all the DeezyMatch main functionalities with minimal explanations. If you want to know more about each module, refer to the relevant part of README (also referenced in this section):
```

Comment:
Maybe, after "with minimal explanations", we could specify that in the quick tour we're just providing examples using the integrated python module, not command line.

Reply:
Done